.github/workflows/cmake.yml

```diff
@@ -1,20 +1,26 @@
-name: CMake
+name: Build and test
 
 on: [push, pull_request]
 
-env:
-  BUILD_TYPE: Release
-
 jobs:
   ubuntu-build:
     name: Build - Ubuntu
     strategy:
       matrix:
-        config:
-          - {os: 'ubuntu-20.04', compiler: {c: gcc, cxx: g++}}
-          - {os: 'ubuntu-22.04', compiler: {c: gcc, cxx: g++}}
-          - {os: 'ubuntu-22.04', compiler: {c: clang, cxx: clang++}}
-    runs-on: ${{matrix.config.os}}
+        os: ['ubuntu-20.04', 'ubuntu-22.04']
+        build_type: [Debug, Release]
+        compiler: [{c: gcc, cxx: g++}]
+        libbacktrace: ['-DVAL_USE_LIBBACKTRACE_BACKTRACE=OFF']
+        include:
+          - os: 'ubuntu-22.04'
+            build_type: Release
+            compiler: {c: clang, cxx: clang++}
+          - os: 'ubuntu-22.04'
+            build_type: Release
+            compiler: {c: gcc, cxx: g++}
+            libbacktrace: '-DVAL_USE_LIBBACKTRACE_BACKTRACE=ON'
```

Comment:
Should the `libbacktrace` be built only with gcc/g++ compilers?

Reply:
Added clang check.